otter/convergence/service.py

```diff
@@ -340,11 +352,7 @@ def convergence_succeeded_servers(scaling_group, group_state, servers, now):
         yield cf_msg('group-status-active',
                      status=ScalingGroupStatus.ACTIVE.name)
     # update servers cache with latest servers
-    yield Effect(
-        UpdateServersCache(
-            scaling_group.tenant_id, scaling_group.uuid, now,
-            [thaw(s.json.set("_is_as_active", True))
-             for s in servers if s.state != ServerState.DELETED]))
+    yield executor.update_cache(scaling_group, servers, None, now)
```

Comment:
The interface of `update_cache` changes with this. It will be nice to document that mentioning clearly the difference between passing `None` vs `[]` (empty list) in `lb_nodes`.